fdbcli/ConsistencycheckCommand.actor.cpp

```diff
@@ -0,0 +1,48 @@
+#include "fdbcli/fdbcli.h"
+
+#include "fdbclient/FDBOptions.g.h"
+#include "fdbclient/IClientApi.h"
+
+#include "flow/Arena.h"
+#include "flow/FastRef.h"
+#include "flow/ThreadHelper.actor.h"
+#include "flow/actorcompiler.h" // This must be the last #include.
+
+using namespace fdb_cli;
+
+ACTOR static Future<bool> consistencycheckCommandActor(Reference<IDatabase> db, std::vector<StringRef> tokens) {
```

Comment:
I'm guessing the casing here is meant to match the `fdbcli` command `consistencycheck`, but I have a feeling this is going to end up being confusing. Can we adopt the normal camel casing for the function and file names (e.g. `consistencyCheckCommandActor`?

Reply:
Yeah, I tried to get it the same as the command name.
Yeah, I agree, I changed this to the normal casing.